implementations/rust/ockam/ockam_api/src/kafka/portal_worker.rs

```diff
@@ -275,6 +277,13 @@ impl KafkaPortalWorker {
             .start_worker(inlet_address.clone(), inlet_worker, AllowAll, AllowAll)
             .await?;
 
+        if let Some((sessions, session_id)) = session {
+            sessions.add_consumer(
+                &outlet_address,
```

Comment:
I don't understand why it is working: we are only allowing outlet worker (aka worker that handles responses from kafka) but we are not explicitly allowing the inlet (aka worker that handles requests to kafka)